www/src/views/starter-showcase/filtered-showcase.js

```diff
@@ -48,25 +48,38 @@ export default class FilteredShowcase extends Component {
       )
     )
 
-    let items = data.allMarkdownRemark.edges,
-      imgs = data.allFile.edges
+    let starters = data.allStartersYaml.edges
+
+    if (starters[0].node.fields.starterShowcase.error) {
```

Comment:
This isn't perfect. Does need to change in some way. Would appreciate thoughts/ideas.

If there's a token, but the request for github data fails for some reason, wouldn't want this error on the built site.

Reply:
Do you think instead of ui messaging, should just display empty values?

Reply:
Done